src/App.php

```diff
@@ -5,16 +5,13 @@
 namespace Friendica;
 
 use Detection\MobileDetect;
+use DOMDocument;
+use DOMXPath;
 use Exception;
-use Friendica\Core\Config;
```

Comment:
Why have you removed these "use" statements? I find it more practical to call "Config::set" than "Core\Config::set" (especially since the backslash is hard to type on German keyboards)

Reply:
Because `Core` is short and `App` uses almost all the `Core\*` classes, so the `use` list was becoming half a screen long. It just is an aesthetic thing for me.